package.json

```diff
@@ -74,6 +74,7 @@
   },
   "devDependencies": {
     "@cloudscape-design/browser-test-tools": "^3.0.4",
+    "@cloudscape-design/build-tools": "^3.0.1",
```

Comment:
Not directly related to your PR, but should we align all `@cloudscape-design` dependencies versions here?

Reply:
Do you mean align them to `^3.0.0`?

Reply:
Yes